setup.py

```diff
@@ -35,7 +35,8 @@ def readme():
         'requests',
         'sh',
         'lxml',
-        'six'
+        'six',
+        'pygithub'
```

Comment:
We do not make VCS modules hard dependencies, but we've made github a hard dependency.
This has to be consistent with current approach.